src/models/monster.js

```diff
@@ -1,15 +1,119 @@
-var mongoose = require('mongoose');
-var Schema = mongoose.Schema;
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
 
-var MonsterSchema = new Schema({
+const MonsterActionDamageType = new Schema ({
+  index: String,
+  name: String,
+  url: String,
+})
```

Comment:
I'm not sure why the CI checks didn't run but this won't past the linting step. You can run `npm run lint` locally to check.

Reply:
Weird that Prettier forces that lack of trailing commas, it's generally accepted as best practice nowadays. Anyway, Done!

Reply:
I've actually been meaning to edit the prettier config to accept trailing commas. I just haven't gotten around to it yet.